bin/node/runtime/src/constants.rs

```diff
@@ -59,11 +59,11 @@ pub mod time {
 	// 1 in 4 blocks (on average, not counting collisions) will be primary BABE blocks.
 	pub const PRIMARY_PROBABILITY: (u64, u64) = (1, 4);
 
-	pub const EPOCH_DURATION_IN_BLOCKS: BlockNumber = 10 * MINUTES;
+	pub const EPOCH_DURATION_IN_SLOTS: BlockNumber = 10 * MINUTES;
 	pub const EPOCH_DURATION_IN_SLOTS: u64 = {
 		const SLOT_FILL_RATE: f64 = MILLISECS_PER_BLOCK as f64 / SLOT_DURATION as f64;
 
-		(EPOCH_DURATION_IN_BLOCKS as f64 * SLOT_FILL_RATE) as u64
+		(EPOCH_DURATION_IN_SLOTS as f64 * SLOT_FILL_RATE) as u64
```

Comment:
This one was already correct as we calculate the number of slots in an epoch based on the number of expected blocks and the slot fill rate (based on the expected block time vs slot duration), i.e. if I have slot duration of 1s but expect block time to be 2s then on average 50% of the slots will be unused. This constants code is more general than the Polkadot one since it started from a point where BABE didn't have secondary slots.

Reply:
Okay, I get what you mean, and I reverted this one.

So that means this PR, unlike the Polkadot one, is not just a rename, it's actually fixing a bug where the incorrect EpochDuration was passed to `pallet_babe` right? pallet babe expects the value passed into be in slots, but we were incorrectly passing the number of actual blocks that are likely to be in the epoch based on the assumed fill rate.

Reply:
No, we were passing `EPOCH_DURATION_IN_SLOTS` for the `EpochDuration`. The only two places where we weren't using `EPOCH_DURATION_IN_SLOTS` (which this PR updates) were regarding staking configuration. Still, given the rest of the constants `EPOCH_DURATION_IN_SLOTS` == `EPOCH_DURATION_IN_BLOCKS` since the `SLOT_FILL_RATE` is 100% :)